### DISCUSSION TITLE
Regression Model Review

### DISCUSSION THREAD

**Discussion by eeinnew** [teamalgoritma/community#120] [Q&A] [answered]
Hello, I have deployed my model but since it is my first time I need the feedback on this `Linear Regression Model`.
here is the link : https://rpubs.com/wenniechangg/life_expectancy

The evaluation is the model overfit with RMSE evaluation and is not Homoscedasticity

thankyou

**Comment by NabiilahArdini** [comment, marked as answer]
Hi @eeinnew ,

We are very excited knowing that you have created your own analytics project using Linear Regression Model! We suggest you to submit the project in the assignment **Learn by Building: Regression Model** in the **Algoritma Academy Google Classroom**. We will provide more thorough feedback from our mentor in there. 

Thank you for your active participation in the class and class assignment!
